assets/js/modules/adsense/components/module/ModuleTopEarningPagesWidget/index.js

```diff
@@ -70,15 +71,46 @@ function ModuleTopEarningPagesWidget( {
 				limit: 10,
 			};
 
+			const report = select( MODULES_ANALYTICS ).getReport( reportArgs );
+
+			let pageTitles;
+
+			let hasLoadedPageTitles = true;
```

Comment:
Same here. Should it be `false` by default?

Reply:
https://github.com/google/site-kit-wp/pull/3958/commits/3cddcca994d8eb6a2e939ac3aca4998081ba16f2